r/rsparkling/tests/run.py

```diff
@@ -0,0 +1,18 @@
+"""
```

Comment:
`tests/run.py` is not a part of rsparkling package, it should be placed outside of R package. Package check already runs every *.R in tests directory. If we want to it run them from gradlew then it should be fine to have run.py outside of rsparkling directory.